src/App.jsx

```diff
@@ -14,33 +17,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABETICALLY = 'alphabet';
+const SORT_FIELD_LENGTH = 'by length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, reverse, { sortField }) {
```

Comment:
You can move this function to a separate file, then import and use it inside a component.